DataCollection/app/src/main/java/com/example/p6/activities/DisplayActivity.java

```diff
@@ -228,19 +247,26 @@ public void onSensorChanged(SensorEvent event) {
         }
     }
 
+    private double getBatteryLevel() {
+        int level = batteryStatus.getIntExtra(BatteryManager.EXTRA_LEVEL, -1);
+        int scale = batteryStatus.getIntExtra(BatteryManager.EXTRA_SCALE, -1);
```

Comment:
Kan man ikke tilføje den her kommentar //scale is the maximum battery percentage

Reply:
Jo det kan man. Det vidste jeg ikke, at det var.
Will do.